Sources/Orbit/Support/Forms/KeyValueField.swift

```diff
@@ -0,0 +1,69 @@
+import SwiftUI
+
+/// A `KeyValue` container with generic content.
+///
+/// - Related components:
+///   - ``InputField``
+///   - ``Text``
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/keyvalue/)
+public struct KeyValueField<Content: View>: View {
```

Comment:
Why not make this be the `KeyValue` and just have a special `init` for the case where content is a `Text`?

Reply:
I wanted to avoid generic content components to be at high level, so that we can put more types inside.